### PR TITLE
Add Chromium versions for XMLHttpRequest API

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adds real values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) for the `XMLHttpRequest` API by mirroring the data.

### CODE REVIEW

File: api/XMLHttpRequest.json [hunk 1]
```diff
@@ -1015,7 +1021,7 @@
                 "version_added": "31"
               },
               "chrome_android": {
-                "version_added": true
+                "version_added": "55"
```
Comment (foolip) [inline api/XMLHttpRequest.json:1024]: What data is this mirroring? I would expect "31" here.

Comment (queengooborg) [reply api/XMLHttpRequest.json]: This was to fix a consistency issue, but another merged PR already updates this data and sets it to "31", which seems more reasonable!

File: api/XMLHttpRequest.json [hunk 2]
```diff
@@ -985,9 +985,15 @@
                 "version_removed": "15"
               }
             ],
-            "opera_android": {
-              "version_added": "42"
-            },
+            "opera_android": [
+              {
+                "version_added": "42"
```
Comment (foolip) [inline api/XMLHttpRequest.json:990]: This doesn't seem right, it doesn't match Opera 18 above. Some mirroring has gone wrong here.

File: api/XMLHttpRequest.json [hunk 3]
```diff
@@ -1038,9 +1044,15 @@
                   "version_removed": "15"
                 }
               ],
-              "opera_android": {
-                "version_added": true
-              },
+              "opera_android": [
+                {
+                  "version_added": "42"
```
Comment (foolip) [inline api/XMLHttpRequest.json:1049]: This also doesn't add up with the above data.

File: api/XMLHttpRequest.json [hunk 4]
```diff
@@ -1083,12 +1095,24 @@
...
+                  "version_added": "18"
+                },
+                {
+                  "version_added": "12",
+                  "version_removed": "15"
+                }
+              ],
+              "opera_android": [
+                {
+                  "version_added": "42"
```
Comment (foolip) [inline api/XMLHttpRequest.json:1109]: This also doesn't add up with the above data.

File: api/XMLHttpRequest.json [hunk 5]
```diff
@@ -1131,12 +1155,24 @@
...
```
Comment (foolip) [inline api/XMLHttpRequest.json:1169]: This also doesn't add up with the above data.

File: api/XMLHttpRequest.json [hunk 6]
```diff
@@ -1188,9 +1224,15 @@
                   "version_removed": "15"
                 }
               ],
-              "opera_android": {
-                "version_added": null
-              },
+              "opera_android": [
+                {
+                  "version_added": "42"
```
Comment (foolip) [inline api/XMLHttpRequest.json:1229]: This also doesn't add up with the above data.

**Review by foolip — CHANGES_REQUESTED** [review]

File: api/XMLHttpRequest.json (hunk 2 above)
Comment (queengooborg) [reply api/XMLHttpRequest.json:990]: Opera Android was originally set to "42" for this feature so I stuck with that.  However, I realize that doesn't really make sense for a feature like this one, so I'm replacing "42" with "18".

**Review by foolip — APPROVED** [review]

**Review by sophiaprincess002 — COMMENTED** [review]
153a95d

**Pull request merged**